en/02_Developer_Guides/14_Files/04_File_Storage.md

```diff
@@ -43,7 +43,7 @@ These records are composite fields which contain sufficient information useful t
 to store, manage, and  publish files. By default this composite field behind this field stores the following details:
 
 | Field name     | Description |
-| ----------     | -----------
+| ----------     | ----------- |
```

Comment:
Unrelated fix to table syntax. It's rendering fine without this for now but other markdown parsers could be more strict if we change our doc renderer in the future.